src/components/Header/Header.jsx

```diff
@@ -1,6 +1,5 @@
-/* eslint-disable no-unused-vars */
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Just like the previous review said kindly solve linters errors intead of disabling them, note that they make your code more maintenable and prevent future bags